packages/ag-charts-community/src/chart/navigator/shapes/rangeSelector.ts

```diff
@@ -30,6 +31,7 @@ export class RangeSelector extends Layer {
 
         this.background.translationX = x;
         this.background.translationY = y;
+        this.markDirty(RedrawType.MAJOR);
```

Comment:
nit: Technically this is redundant because `this` is an ancestor of `this.background` and therefore `this.background.translationY = y` will set `this._dirty = RedrawType`. However I've just added it as a safety measure because redundant `markDirty` calls do not have any side-effects.